doc/dev_guide/internals/box_protocol.rst

```diff
@@ -719,6 +735,33 @@ IPROTO_SQL_TEXT (0x40) + statement-text (string) if executing an SQL string.
 Thus the IPROTO_PREPARE map item is the same as the first item of the
 :ref:`IPROTO_EXECUTE <box_protocol-execute>` body.
 
+..  _box_protocol-begin:
+
+**IPROTO_BEGIN** = 0x0e.
+
+This is for starting a transaction.
+Typically the header will include IPROTO_STREAM_ID.
+The body is: nothing.
+See the later section :ref:`Binary protocol -- streams <box_protocol-streams>`.
+
+..  _box_protocol-commit:
+
+**IPROTO_COMMIT** = 0x0f.
+
+This is for starting a transaction.
```

Comment:
IPROTO_COMMIT used to commit transaction and IPROTO_ROLLBACk used to rollback it not for starting it.

Reply:
Yes, fixed.